guides/ageur-treasury.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Using agEUR as a treasury asset for your DAO or company
+---
+
+# agEUR as a treasury asset
+
+As we described [here](/guides/ageur-power-usecase.md), agEUR can be used by companies and DAOs to pay their contributors. But there is more!
```

Comment:
Still sounds like an article, we need to make it more like a guide. Does not imply changing much